test/specs/validate-spec/invalid/3.x/duplicate-operation-ids.yaml

```diff
@@ -0,0 +1,17 @@
+openapi: 3.0.0
+info:
+  version: "1.0.0"
+  title: Invalid API
+
+paths:
+  /users:
+    get:
+      operationId: users
+      responses:
+        default:
+          description: hello world
+    post:
+      operationId: users          # <---- duplicate
```

Comment:
This library has long had validation support for this kind of thing for Swagger 2.0 specs, this work will extend that to OAS 3.0+.